rudder-core/src/main/scala/com/normation/rudder/repository/jdbc/ExpectedReportsJdbcRepository.scala

```diff
@@ -260,15 +277,15 @@ class FindExpectedReportsJdbcRepository(
             // also, get policy mode from lib - if the directive is missing, we don't know what to do, but it's
             // not more false than if the value changed since expected reports were written.
             val mode = directivesLib.allDirectives.get(directiveId).flatMap( _._2.policyMode)
-            DirectiveExpectedReports(directiveId, mode, lines.map( x =>
-              ComponentExpectedReport(x.component, x.cardinality, x.componentsValues, x.unexpandedCptsValues)
-            ).distinct /* because we have the cardinality to deals with mutltiplicity */ )
+            DirectiveExpectedReports(directiveId, mode, false, lines.map( x =>
```

Comment:
What if we a System directive ? can't we get bak isSystem value when getting mode just above ?